<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoukaiTabForm</class>
 <widget class="QWidget" name="YoukaiTabForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>GENERAL</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="num1Label">
            <property name="text">
             <string>NUM1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="num1SB">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="num2Label">
            <property name="text">
             <string>NUM2</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="num2SB">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="youkaiLabel">
            <property name="text">
             <string>TYPE_OF_YOKAI</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="youkaiCB">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="nicknameLabel">
            <property name="text">
             <string>NICKNAME</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="nicknameEdit">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="levelLabel">
            <property name="text">
             <string>LEVEL</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="levelSB">
            <property name="minimum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="IDLabel">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="IDEdit">
            <property name="inputMask">
             <string notr="true">HHHHHHHH</string>
            </property>
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="atkLVLabel">
            <property name="text">
             <string>ATK_LEVEL</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="inspiritLVLabel">
            <property name="text">
             <string>INSPR_LEVEL</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="soultimateLVLabel">
            <property name="text">
             <string>SOULTIMATE_LEVEL</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="atkLVSB">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="inspiritLVSB">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="soultimateLVSB">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="loafLabel">
            <property name="text">
             <string>PERSONAL_LOAF</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="loafSB">
            <property name="maximum">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="AILabel">
            <property name="text">
             <string>PERSONAL_AI</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="AICB"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="trainingLabel">
            <property name="text">
             <string>TRAINING_COUNT</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="trainingSB">
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>IV</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="IV_HPLabel">
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="IV_HPSB">
            <property name="maximum">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="IV_StrLabel">
            <property name="text">
             <string>STR</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="IV_StrSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="IV_SprLabel">
            <property name="text">
             <string>SPR</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="IV_SprSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="IV_DefLabel">
            <property name="text">
             <string>DEF</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="IV_DefSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="IV_SpdLabel">
            <property name="text">
             <string>SPD</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="IV_SpdSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>SPORTS_CLUB</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="SC_HPLabel">
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="SC_StrLabel">
            <property name="text">
             <string>STR</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="SC_SprLabel">
            <property name="text">
             <string>SPR</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="SC_DefLabel">
            <property name="text">
             <string>DEF</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="SC_SpdLabel">
            <property name="text">
             <string>SPD</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="SC_HPSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="SC_StrSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="SC_SprSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="SC_DefSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="SC_SpdSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="flags">
       <property name="title">
        <string>FLAGS</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="flag4CB">
            <property name="text">
             <string>IS_SPECIAL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="flag5CB">
            <property name="text">
             <string>IS_NEW</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="flag6CB">
            <property name="text">
             <string notr="true">6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="flag7CB">
            <property name="text">
             <string>IS_CHEATED</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>83</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="youkaiNumApplyB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>UPDATE_INDEX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateIDButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>UPDATE_ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fixIVButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>FIX_IV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="allPoseB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ALL_WINPOSES</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
